.github/workflows/gradle-data-capturing-samples-verification.yml

```diff
@@ -42,3 +42,10 @@ jobs:
       - name: Run Gradle build
```

Comment:
I would precise that this is the Gradle sample step

Reply:
done